src/components/cards/Cards.js

```diff
@@ -1,16 +1,17 @@
 import "./Cards.css";
-import CardHermione from "../CardHermione/CardHermione";
-import CardCedric from "../CardCedric/CardCedric";
-import CardDraco from "../CardDraco/CardDraco";
-import CardHarry from "../CardHarry/CardHarry";
-import CardRon from "../CardRon/CardRon";
-import harryPicture from "../img/harry.png";
-import dracoPicture from "../img/draco.png";
-import hermionePicture from "../img/hermione.png";
-import cedricPicture from "../img/cedric.png";
-import ronPicture from "../img/ron.png";
-import data from "../data.json";
+// import CardHermione from "../CardHermione/CardHermione";
+// import CardCedric from "../CardCedric/CardCedric";
+// import CardDraco from "../CardDraco/CardDraco";
+// import CardHarry from "../CardHarry/CardHarry";
+// import CardRon from "../CardRon/CardRon";
+// import harryPicture from "../img/harry.png";
+// import dracoPicture from "../img/draco.png";
+// import hermionePicture from "../img/hermione.png";
+// ((import cedricPicture from "../img/cedric.png";
+// import ronPicture from "../img/ron.png";
```

Comment:
Please remove this commented out code - you don't use it anymore 🙂

Reply:
Thx for the heads up!

ad1) I actually know, but still done wrong... 🙈

ad2) I totally agree. I just wasn't sure if the files were needed for todays class and wanted to be able to restore them in seconds if needed. Same with the commented out code. I'll delete/remove both ✌️🙂